bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -50,16 +62,56 @@ spec:
           echo "$(date -Iseconds) - starting ovs-daemons"
           chown -R openvswitch:openvswitch /run/openvswitch
           chown -R openvswitch:openvswitch /etc/openvswitch
+          if [[ -d /run/openvswitch ]]; then
+            ls -al /run/openvswitch
+          fi
+          if [[ -d /var/run/openvswitch ]]; then
+            ls -al /var/run/openvswitch
+          fi
+          if [[ -f /var/run/openvswitch/ovs-vswitchd.pid ]] ; then
+            cat /var/run/openvswitch/ovs-vswitchd.pid
+          fi
+          if [[ -f /var/run/openvswitch/ovsdb-server.pid ]] ; then
+            cat /var/run/openvswitch/ovsdb-server.pid
+          fi
           function quit {
```

Comment:
Yikes. This needs to be moved to the "non-systemd case"

Reply:
This is all debug code

Reply:
not the `kill -9`...